src/test/java/org/apache/commons/imaging/formats/tiff/TiffSubImageTest.java

```diff
@@ -79,13 +73,10 @@ public void testBadSubImage()  throws ImageReadException, IOException{
     private void processBadParams(final File target, final int x, final int y, final int width, final int height, final String comment) throws IOException{
         try{
             final TiffImagingParameters params = new TiffImagingParameters();
-            params.setSubImageX(x);
-            params.setSubImageY(y);
-            params.setSubImageWidth(width);
-            params.setSubImageHeight(height);
+            params.setSubImage(x, y, width, height);
             final BufferedImage image = Imaging.getBufferedImage(target, params);
             fail("Reading TIFF sub-image failed to detect bad parameter: "+comment);
-        }catch(final ImageReadException ire){
+        }catch(final ImageReadException | IllegalArgumentException ire){
```

Comment:
See comment above, if we use `ImageReadException`, then we don't need this extra exception here.